test/functional/lua/vim_spec.lua

```diff
@@ -794,6 +794,16 @@ describe('lua stdlib', function()
           pcall_err(exec_lua, "vim.fn.nvim_get_current_line()"))
   end)
 
+  it('vim.fn can be called in fast events (if function is allowed)', function()
+    exec_lua([[
+      vim.loop.fs_stat('.', function()
+        vim.g.fnres = vim.fn.iconv('hello', 'utf-8', 'utf-8')
```

Comment:
This passes because of the above issue, I guess

Reply:
Yeah.

This is the simplest way I could think to get into a fast event. Using a timer might be better though.